doc/release-notes.md

```diff
@@ -36,6 +36,13 @@ No attempt is made to prevent installing or running the software on Windows XP,
 you can still do so at your own risk, but do not expect it to work: do not
 report issues about Windows XP to the issue tracker.
 
+From 0.13.1 onwards OS X 10.7 is no longer supported. Issues with the libc++
```

Comment:
@laanwj Yes. I just did a fresh download/run of 0.13.0 on a 10.7 vm to verify. Broken.